File: js/charCount.js
exports.charCount = function(string) {
    let obj={}
    let resArr;
    string= string.toString().replace(/[^a-zA-Z0-9]/g,"").toLowerCase().split("").sort()
    string.forEach(value => {
        if (obj[value]){
        obj[value] +=1
    }else{
        obj[value]=1
    }})
    resArr= Object.entries(obj)
    resArr.sort((a, b)=> {
    return b[1] - a[1]
})
    return resArr
};

File: python/char_count.py
def char_count(str):
  obj= {}
  res_arr=[]
  input_arr= list(input.replace(" ", "").lower())
  for value in input_arr:
    if value in obj.keys():
      obj[value]+=1
    else:
      obj[value]=1
  for key in obj:
    res_arr.append([key,obj[key]])
  
  return sorted(res_arr, key=lambda x : (-x[1], x[0]))
         